cmd/create/project.go

```diff
@@ -0,0 +1,96 @@
+package create
+
+import (
+	"context"
+	"fmt"
+	"io/ioutil"
+
+	"github.com/lyft/flyteidl/gen/pb-go/flyteidl/admin"
+	"gopkg.in/yaml.v2"
+
+	cmdCore "github.com/lyft/flytectl/cmd/core"
+)
+
+const (
+	projectShort = "Create project resources"
+	projectLong  = `
+Create the projects.(project,projects can be used interchangeably in these commands)
```

Comment:
We can mention somewhere here which are the mandatory fields and which aren't.

Reply:
can you give me a example